ui/lib/sync/addon/components/secrets/page/overview.hbs

```diff
@@ -149,21 +149,17 @@
       </h2>
     </OverviewCard>
     <OverviewCard
-      @cardTitle="Total sync associations"
-      @subText="The number of secrets with a configured sync destination. One secret synced to two unique destinations will count as two associations."
+      @cardTitle="Total secrets"
+      @subText="The total number of secrets synced from Vault."
       {{!-- * sync clients are unavailable during SYNC BETA (1.16.0), planned for 1.16.1 release 
-      @subText="Total sync associations that count towards client count"
```

Comment:
removed since this subtext will no longer be accurate since the latest decision is that sync clients = vault synced secrets (this number)